app/cdap/components/ConnectionList/Components/ConnectionTabs/Components/RenderLabel/index.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import TabLabelCanBrowse from 'components/ConnectionList/Components/TabLabelCanBrowse';
+import TabLabelCanSample from 'components/ConnectionList/Components/TabLabelCanSample';
+import React from 'react';
+import { IRenderLabelProps } from 'components/ConnectionList/Components/ConnectionTabs/types';
+
+export default function({
+  columnIndex,
+  connectorType,
+  connectionIdProp,
+  toggleLoader,
+  setIsErrorOnNoWorkSpace,
+}: IRenderLabelProps) {
```

Comment:
move the interface to this file and delete from common types file
export from here and reuse if necessary

Reply:
Done